contracts/optimistic-ethereum/OVM/chain/OVM_CanonicalTransactionChain.sol

```diff
@@ -1007,14 +1008,14 @@ contract OVM_CanonicalTransactionChain is iOVM_CanonicalTransactionChain, Lib_Ad
      * @param _prevContext The previously validated batch context.
      * @param _nextContext The batch context to validate with this call.
      * @param _nextQueueIndex Index of the next queue element to process for the _nextContext's subsequentQueueElements.
-     * @param _queueLength The length of the queue at the start of the batchAppend call.
+     * param _queueLength The length of the queue at the start of the batchAppend call.
      * @param _queueRef The storage container for the queue.
      */
     function _validateNextBatchContext(
         BatchContext memory _prevContext,
         BatchContext memory _nextContext,
         uint40 _nextQueueIndex,
-        uint40 _queueLength,
```

Comment:
Seems like this variable could just be removed entirely from the params here. That might be the better way to clear out this compiler warning.

Reply:
Ah yes, confirming that this is vestigial now with https://github.com/ethereum-optimism/contracts/pull/307/ pulling the logic into `_validateFinalBatchContext`.